xcp_d/workflow/anatomical.py

```diff
@@ -647,25 +648,13 @@ def init_anatomical_wf(
                 )
             )
 
-            fs_L2fsLR = str(
-                get_template(
-                    template="fsaverage",
-                    hemi="L",
-                    density="164k",
-                    desc="std",
-                    suffix="sphere",
-                )
-            )
+            fs_L2fsLR = pkgrf('xcp_d',
+                              ('data/standard_mesh_atlases/fs_L/'
+                               'fs_L-to-fs_LR_fsaverage.L_LR.spherical_std.164k_fs_L.surf.gii'))

```

Comment:
Why drop templateflow?

Reply:
Answered in today's meeting- templateflow is missing one of the spheres that the DCAN lab wants. Do you know which one?

Reply:
The sphere I couldn't find in TemplateFlow is the `fs_L-to-fs_LR_fsaverage.L_LR.spherical_std.164k_fs_L.surf.gii` (and the corresponding RH sphere). It's possible I didn't look hard enough for it though. I assumed the `tpl-fsaverage_hemi-L_den-164k_desc-std_sphere.surf.gii` was the same but was mistaken, hence this change.

Reply:
Wow, that naming convention is... dense. Is it the transform of the fsaverage left hemisphere to fsLR at 164k density?